lib/hooks/createHighlights.ts

```diff
@@ -6,7 +6,7 @@ interface CreateHighlightsProps {
   title?: string;
   highlight: string;
   shipped_at?: Date;
-  type: "issue" | "pull_request";
+  type: "issue" | "pull_request" | "blog_post";
```

Comment:
This is duplicate code, could we please reuse this type `HighlightType`

Reply:
Thanks!